### ISSUE TITLE
Fine-tune model hyperparameters, such as learning rate, batch size, etc., to optimize model performance on the validation set.

### ISSUE THREAD

**Issue by otman-dev** [AI-ForestFireDetection/AI-Yolo#22]
Related to UserStory #7
